etcdmain/help.go

```diff
@@ -70,6 +70,12 @@ member flags:
 		maximum number of operations permitted in a transaction.
 	--max-request-bytes '1572864'
 		maximum client request size in bytes the server will accept.
+	--grpc-keepalive-min-time '5s'
+		minimum duration interval that a client should wait before pinging server.
+	--grpc-keepalive-interval '2h'
```

Comment:
2h seems too high. probably 5 minutes or something? is there a recommended vault from gRPC-go?

Reply:
gRPC Go defaults to 2h. Should etcd use 5min?

Reply:
2h is fine if it is gRPC's default.